spec/requests/users/create_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+# Test suite for POST /auth
+describe 'POST /auth', type: :request do
+  # valid payload
+  let(:valid_params) { attributes_for(:user) }
+  let(:invalid_params) { attributes_for(:user, password: '1234') }
+
+  context 'when the request is valid' do
```

Comment:
Hey, just to be sure it would be nice to add a new test under this context just to check that `sending all valid params the resultant user has those attributes` just to make sure strong params where added to the default devise params